### PR TITLE
[PIL-577][SM] Updated read me file

### PR DESCRIPTION
**Pull request by srikanthhmrc**

### CODE REVIEW

**Review by annemonica5 — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,38 +1,185 @@
 
 # pillar2-acceptance-tests
 
-Acceptance tests for Pillar2 Application Features
-
+####Acceptance tests for Pillar2 Application Features
```
Comment (jassalrichy) [inline README.md:4]: Needs a space after the #

File: README.md [hunk 2]
```diff
@@ -1,38 +1,185 @@
...
+* https://www.staging.tax.service.gov.uk/report-pillar2-top-up-taxes/contact-details/check-answers
+* https://www.staging.tax.service.gov.uk/report-pillar2-top-up-taxes/review-submit/check-answers
+* https://www.staging.tax.service.gov.uk/report-pillar2-top-up-taxes/review-submit/confirmation
+* https://www.staging.tax.service.gov.uk/report-pillar2-top-up-taxes/pillar2-top-up-tax-home
+* https://www.staging.tax.service.gov.uk/report-pillar2-top-up-taxes/manage-account/contact-details/summary
+* https://www.staging.tax.service.gov.uk/report-pillar2-top-up-taxes/manage-account/further-details/change-group-status
+* https://www.staging.tax.service.gov.uk/report-pillar2-top-up-taxes/manage-account/further-details/change-accounting-period
+* https://www.staging.tax.service.gov.uk/report-pillar2-top-up-taxes/manage-account/account-details/summary
+
+####The structure of the project is as follows:
```
Comment (jassalrichy) [inline README.md:58]: Needs a space after the #

File: README.md [hunk 3]
```diff
@@ -1,38 +1,185 @@
...
+└── run_zap_tests_local.sh       # Script file to run security tests on local
+```
 
+It is built using the following:
+- Cucumber(BDD),
+- Selenium-WebDriver,
+- SBT (build tool),
+- Scala
+
+###Getting started
```
Comment (jassalrichy) [inline README.md:125]: Needs a space after the #

File: README.md [hunk 4]
```diff
@@ -1,38 +1,185 @@
...
     
     IntelliJ 
     ---------
-    
-    Create a new JUnit task and set the class to uk.gov.hmrc.test.ui.cucumber.runner
-    Change use "classpath of module" to pillar2-acceptance-tests     
+    - Create a new JUnit task and set the class to uk.gov.hmrc.test.ui.cucumber.runner
+    - Change use "classpath of module" to pillar2-acceptance-tests     
+
+###Source code formatting
```
Comment (jassalrichy) [inline README.md:154]: Needs a space after the #

File: README.md [hunk 5]
```diff
@@ -1,38 +1,185 @@
 
 # pillar2-acceptance-tests
 
-Acceptance tests for Pillar2 Application Features
-
+####Acceptance tests for Pillar2 Application Features
+This repository is the automated acceptance tests for Pillar2 Application which include below pages:
```
Comment (jassalrichy) [inline README.md:5]: Can we change this to something like:

These are the acceptance tests for the Pillar 2 (Global Minimum Tax) service. They test the user journey for Registering, Subscription and Maintaining Pillar 2 tax for MNEs. The service is tested on the Jenkins build pipeline for tests that run through the full journey pages included below:

**Review by jassalrichy — APPROVED** [review]

**Pull request merged**